addons/mail/static/src/core/web/recipients_popover.js

```diff
@@ -0,0 +1,29 @@
+import { useService } from "@web/core/utils/hooks";
+
+import { onWillStart } from "@odoo/owl";
+import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
+
+export class RecipientsPopover extends AvatarCardPopover {
+    static template = "mail.RecipientsPopover";
+    static props = {
+        ...AvatarCardPopover.props,
+        viewProfileBtnOverride: { type: Function },
+    };
+
+    setup() {
+        this.orm = useService("orm");
+        onWillStart(async () => {
+            [this.user] = await this.orm.read("res.partner", [this.props.id], this.fieldNames);
+            this.user.email = this.user.email_normalized;
```

Comment:
What does that do ?

Reply:
Why is it called this.user when we read a res partner ?

Reply:
The value read in the template of the Component is 'this.user'. So in order to not have to rewrite the whole template I assigned to the value already present.
If I made the Component from scratch I would've used partner. But since I am constrained to the values already present I had to use this.

Reply:
Ok :)